### PR TITLE
Added investors section

### PR DESCRIPTION
**Pull request by shrinathprabhu**
1) Added investor section
2) Updated responsive position of investors
3) Added investor images
4) Added investor description on click of investor
5) Added mock description and orbs for investor

### CODE REVIEW

File: components/home/Investors.vue [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+</template>
+
+<script>
+import AppSectionDescriptor from '../AppSectionDescriptor.vue'
+import VButton from '../lib/VButton.vue'
+import VContainer from '../lib/VContainer.vue'
+import VHeading from '../lib/VHeading.vue'
+import VImage from '../lib/VImage.vue'
+import VStack from '../lib/VStack.vue'
+import VText from '../lib/VText.vue'
```
Comment (punaifang) [inline components/home/Investors.vue:81]: Please use root relative imports for internal modules.

```
import AppSectionDescriptor from '~/components/AppSectionDescriptor`
```

File: components/home/Investors.vue [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+
+export default {
+  components: {
+    VContainer,
```
Comment (punaifang) [inline components/home/Investors.vue:85]: Sort components alphabetically.

**Review by punaifang — CHANGES_REQUESTED** [review]
Please move the image assets to `assets/images`.

File: components/home/Investors.vue (hunk 1 above)
Comment (shrinathprabhu) [reply components/home/Investors.vue]: @punaifang created root relative imports for internal modules

File: components/home/Investors.vue (hunk 2 above)
Comment (shrinathprabhu) [reply components/home/Investors.vue]: @punaifang sorted components alphabetically

**Pull request merged**